gdnative-core/src/core_types/vector3.rs

```diff
@@ -1,4 +1,14 @@
-use crate::core_types::Vector3;
+use approx::relative_eq;
+use glam::Vec3A;
```

Comment:
@toasteater Should `Vec3A` be used (SIMD support) or `Vec3` (none)?
Both are `#[repr(transparent)]`.

Same should probably be done for 2D vectors.

Reply:
Since we are converting to `glam` vectors by value, there should be no soundness issue with using `Vec3A`/`Vec2A`. The representation of those types do not matter. If we were converting by reference, then it would be unsound to use the SIMD variants since they have additional padding and alignment.

Performance-wise I *think* it should be fine / insignificant. `Vec3A::new` should just compile to a load into the SIMD register.

Reply:
Makes sense, then we should also use `Vec2A`, not `Vec2` (made a comment above).

Reply:
There is no `Vec2A` though.

Reply:
Ah, that explains 🙂 then good as is!